source/viz.md

```diff
@@ -504,14 +504,11 @@ neither of the variables here have a natural order to them.
 So a scatter plot is likely to be the most appropriate
 visualization. Let's create a scatter plot using the `altair`
 package with the `waiting` variable on the horizontal axis, the `eruptions`
-variable on the vertical axis, and the `mark_point` geometric object.
-By default, `altair` draws only the outline of each point. If we would
-like to fill them in, we pass the argument `filled=True` to `mark_point`. In
-place of `mark_point(filled=True)`, we can also use `mark_circle`.
+variable on the vertical axis, and `mark_point` as the graphical mark.
```

Comment:
I gotta say: it's totally insane to me that altair treats "point" as "not-filled-in circle", and "circle" as "filled disk". I think the vast majority of people would consider a "point" to be a "dot" or "small filled circle", and "circle" to mean "not-filled-in circle". Blech